us.zoom.Zoom.appdata.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>us.zoom.Zoom.desktop</id>
+  <name>Zoom</name>
+  <project_license>LicenseRef-proprietary</project_license>
+  <developer_name>zoom.us</developer_name>
+  <summary>Video Conferencing, Web Conferencing, Webinars, Screen Sharing</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://zoom.us</url>
+  <description>
+    <p>Award winning Zoom brings video conferencing, online meetings and group messaging into one easy-to-use application.</p>
+    <p>One consistent enterprise experience for all use cases</p>
```

Comment:
Perhaps these should be an unordered list rather than one sentence paragraphs?

Reply:
Yeah, that's fair.